tools/execution_environments/ee-multicloud-public/install_binaries.sh

```diff
@@ -24,9 +37,8 @@ rm bw bw.zip
 
 
 # AWS CLI
-
 aws_version=2.4.23
-curl -s -L "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-${aws_version}.zip" \
+curl -s -L "https://awscli.amazonaws.com/awscli-exe-linux-$(uname -m)-${aws_version}.zip" \
```

Comment:
shouldn't you be using $ARCH that you set earlier?

Reply:
@fridim didn't work with arm, that's why I'm using uname there